DependencyInjection/CmfSeoExtension.php

```diff
@@ -208,23 +211,25 @@ private function loadAlternateLocaleProvider($config, ContainerBuilder $containe
             return;
         }
 
-        $alternateLocaleProviderDefinition = $container->findDefinition($alternateLocaleProvider);
-        if ($this->contentListenerEnabled) {
-            $container
-                ->findDefinition('cmf_seo.event_listener.seo_content')
-                ->addMethodCall(
-                    'setAlternateLocaleProvider',
-                    array($alternateLocaleProviderDefinition)
-                );
-        }
-        if ($this->phpcrProviderEnabled) {
-            $container
-                ->findDefinition('cmf_seo.sitemap.phpcr_provider')
-                ->addMethodCall(
-                    'setAlternateLocaleProvider',
-                    array($alternateLocaleProviderDefinition)
-                )
-            ;
+        if ($alternateLocaleProvider) {
```

Comment:
we already returned above if $alternateLocaleProvider is not set, so this check can be removed and indention be reverted.